dashboards-observability/public/components/notebooks/components/create.tsx

```diff
@@ -0,0 +1,148 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+/* eslint-disable react-hooks/exhaustive-deps */
+import {
+    EuiButton,
+    EuiFieldText,
+    EuiFlexGroup,
+    EuiFlexItem,
+    EuiForm,
+    EuiFormRow,
+    EuiHorizontalRule,
+    EuiPage,
+    EuiPageBody,
+    EuiPageContent,
+    EuiPageContentHeader,
+    EuiPageContentHeaderSection,
+    EuiPageHeader,
+    EuiPageHeaderSection,
+    EuiSpacer,
+    EuiTitle,
+    EuiToolTip,
+  } from '@elastic/eui';
+import React, { useEffect, useState } from 'react';
+import "../../../../public/components/application_analytics/app_analytics.scss"
+import { ChromeBreadcrumb } from '../../../../../../src/core/public';
+
+  interface CreateNotebookProps {
+    existingNotebookId: string;
+    parentBreadcrumb: ChromeBreadcrumb;
+    setBreadcrumbs: (newBreadcrumbs: ChromeBreadcrumb[]) => void;
+    createNotebook: (name: string) => void;
+    renameNotebook: (name: string, id: string) => void
+  }
+  
+  export const CreateNotebook = (props: CreateNotebookProps) => {
```

Comment:
Let's change the name here  `CreateNotebook` -> `CreateNotebookPage` Maybe? We already have `createNotebook` function. This would be confusing for other developers if not changed.

Reply:
Sure. Changed in [1e6978c](https://github.com/opensearch-project/dashboards-observability/pull/1308/commits/1e6978c45b041cd0bf544e0b86f6a5265b30e60a)